test/test_remap.py

```diff
@@ -154,9 +154,137 @@ def test_edge_centers_remapping(self):
+
+    def test_remap_center_nodes(self):
+        """Test remapping to center nodes."""
+
+        # datasets to use for remap
+        dataset = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+        destination_uxds = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+
+        data_on_face_centers = dataset['v1'].inverse_distance_weighted_remap(
+            destination_uxds.uxgrid, remap_to="face centers")
+
+        assert not np.array_equal(dataset['v1'], data_on_face_centers)
+
+    def test_remap_nodes(self):
+        """Test remapping to nodes."""
+
+        # datasets to use for remap
+        dataset = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+        destination_uxds = ux.open_dataset(gridfile_geoflow, dsfile_v1_geoflow)
+
+        data_on_nodes = dataset['v1'].inverse_distance_weighted_remap(
+            destination_uxds.uxgrid, remap_to="nodes")
+
+        assert not np.array_equal(dataset['v1'], data_on_nodes)
+
+    def test_cartesian_remap_to_nodes(self):
+        """Test remapping using cartesian coordinates using nodes."""
+
+        # triangle with data on nodes
+        source_verts = np.array([(0.0, 0.0, 1.0), (0.0, 1.0, 0.0),
+                                 (1.0, 0.0, 0.0)])
+        source_data = [1.0, 2.0, 3.0]
+
+        # open the source and destination grids
+        source_grid = ux.open_grid(source_verts)
+        destination_grid = ux.open_grid(source_verts)
+
+        # create the first destination data using two k neighbors
+        destination_data_neighbors_2 = _inverse_distance_weighted_remap(
+            source_grid,
+            destination_grid,
+            source_data,
+            remap_to="nodes",
+            coord_type="cartesian",
+            k_neighbors=2)
+
+        # create the second destination data using one k neighbor
+        destination_data_neighbors_1 = _inverse_distance_weighted_remap(
+            source_grid,
+            destination_grid,
+            source_data,
+            remap_to="nodes",
+            coord_type="cartesian",
+            k_neighbors=1)
+
+        # two different k_neighbor remaps are different
+        assert not np.array_equal(destination_data_neighbors_1,
+                                  destination_data_neighbors_2)
+
+    def test_remap_return_types(self):
+        """Tests the return type of the `UxDataset` and `UxDataArray`
+        implementations of Inverse Distance Weighted."""
+
+        source_data_paths = [
+            dsfile_v1_geoflow, dsfile_v2_geoflow, dsfile_v3_geoflow
+        ]
+        source_uxds = ux.open_mfdataset(gridfile_geoflow, source_data_paths)
+        destination_uxds = ux.open_dataset(gridfile_CSne30,
+                                           dsfile_vortex_CSne30)
+
+        remap_uxda_to_grid = source_uxds['v1'].inverse_distance_weighted_remap(
+            destination_uxds.uxgrid)
+
+        assert isinstance(remap_uxda_to_grid, UxDataArray)
+
```

Comment:
How about adding `assert len(remap_uxda_to_uxda.data_vars) == 1`here and for cases that follow?